cultcargo/genesis/wsclean/wsclean-base.yml

```diff
@@ -16,99 +16,412 @@ lib:
           nom_de_guerre: name
           required: true
         column:
+          info: Default is CORRECTED_DATA if it exists, otherwise DATA will be used.
           dtype: str
           nom_de_guerre: data-column
+        model-column:
+          info: Column to which the predicted data is written. Default is MODEL_DATA.
+          dtype: str
+        model-storage-manager:
+          info: If a new model column needs to be written, create the model using this storage manager. Supported types are "default" and "stokes-i".
+          dtype: str
+          choices: [default, stokes-i]
         nchan:
           info: Channels out
           dtype: int
           nom_de_guerre: channels-out
         deconvolution-channels:
-          info: Channels to use in deconvolution
+          info: Decrease the number of channels as specified by -channels-out to the given number for deconvolution. Only possible in combination with one of the -fit-spectral options.
           dtype: int
         channel-range:
           dtype: List[int]
           policies:
             repeat: list
+        channel-division-frequencies:
+          info: Split the bandwidth at the specified frequencies (in Hz) before the normal bandwidth division is performed.
+          dtype: List[float]
+          policies:
+            repeat: list
+        gap-channel-division:
+          info: In case of irregular frequency spacing, this option can be used to not try and split channels to make the output channel bandwidth similar, but instead to split largest gaps first.
+          dtype: bool
         pol:
           dtype: Union[str, List[str]]
           element_choices: [I,Q,U,V,QU,QUV,IQ,IV,IQU,IQUV,XX,XY,YX,YY,LL,LR,RL,RR]
           policies:
             repeat: ","
         join-polarizations:
+          info: Perform deconvolution by searching for peaks in the sum of squares of the polarizations, but subtract components from the individual images. Only possible when imaging two or four Stokes or linear parameters. Default is off.
           dtype: bool
         link-polarizations:
+          info: Links all polarizations to be cleaned from the given list - components are found in the given list, but cleaned from all polarizations.
           dtype: Union[str, List[str]]
           element_choices: [I,Q,U,V,QU,QUV,IQ,IV,IQU,IQUV,XX,XY,YX,YY,LL,LR,RL,RR]
           policies:
             repeat: ","
         intervals-out:
+          info: Number of intervals to image inside the selected global interval. Default is 1.
           dtype: int
+        interval:
+          info: Only image the given time interval. Indices specify the timesteps, end index is exclusive. Default is image all time steps.
+          dtype: List[int]
```

Comment:
I think this should be a ``Tuple[int, int]`` (same for ``channels`` above, which I didn't do right either).